mechanisms/generate_catalogue

```diff
@@ -116,7 +116,17 @@ const mechanism_catalogue& global_${catalogue}_catalogue() {
     return cat;
 }
 
-} // namespace arb''')
+} // namespace arb
+
+#ifdef STANDALONE
+extern "C" {
+    const arb::mechanism_catalogue& get_catalogue() {
```

Comment:
FMI: function signatures/variables of C++ types in an extern block makes them compiler (version) specific, right?

Reply:
It's ill defined and produces a warning I got rid of in the latest round of polish.